docs/javascript/v2/how-to-guides/extend-capabilities/plugins/effects-virtual-background.mdx

```diff
@@ -242,4 +242,32 @@ The full implementation can be viewed in the [roomkit-react package](https://git
 
 ## Supported Browsers
 
-Effects virtual background is supported on Safari, Firefox and Chromium based browsers.<br/>
+Effects virtual background is supported on Safari (v16.5.1+), Firefox and Chromium based browsers.<br/>
+
+## Known issues
+
+Effects is not supported in Safari versions older than v17. It is recommended to use the HMSVBPlugin instead. This conditional usage can be implemented by importing the effects plugin after checking the browser and version through the useragent string.
```

Comment:
this and the above is contradicting. I think it is not just the safari version. it might be a combination of safari and macos versions

Reply:
we also make an example of using parsedUserAgent for this